TOMP-API.yaml

```diff
@@ -2255,7 +2255,7 @@ components:
         number:
           description: phone number. In case of international usage, always provide the country code.
           type: string
-          pattern: '^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\./0-9]*$'
+          pattern: '^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\.0-9]*$'
           example: +31-48934758 or +(0075)-834923384 or 020 1234 1234
```

Comment:
This example is also wrong. If we want to show multiple examples we need to do
```yaml
examples:
    withCountryCodeAndDash:
        value: +31-48934758
    withCountryCodeAndParentheses:
        value: +(0075)-834923384
    withLeadingZero:
        value: 020 1234 1234
```

Reply:
wow! I didn't knew this possibility (syntax). I've changed it (btw it is example: not examples:).

Reply:
It is examples for multiple examples. https://swagger.io/docs/specification/adding-examples/

Reply:
![image](https://user-images.githubusercontent.com/10400054/98926283-65d5a180-24d7-11eb-9d07-8733fa65079d.png)
hmmm. The editor.swagger.io reports an error.

Reply:
This seems a but in the editor.

Reply:
O no, it's not a bug.. This 'examples' keyword is only allowed in the parameters part. Not in the object properties..

Reply:
Oh, that's too bad :/